widget/playground/src/store/config.ts

```diff
@@ -206,18 +194,34 @@ export const useConfigStore = createSelectors(
               }
             }
           }),
-        onChangeColors: (name, mode, color) =>
+        onChangeColors: ({ name, mode, color, singleTheme, resetColors }) =>
           set((state) => {
-            console.log({ color });
-
-            if (state?.config?.theme?.colors) {
-              state.config.theme.colors = {
-                ...state.config.theme.colors,
-                [mode]: {
-                  ...state?.config?.theme.colors[mode],
-                  [name]: color,
-                },
-              };
+            if (state.config?.theme?.colors) {
+              let themes = { ...state.config.theme, singleTheme };
+              if (resetColors) {
```

Comment:
Please add a comment in this section to improve code clarity.

Reply:
I added